src/test/java/com/fasterxml/jackson/databind/ser/filter/JsonIncludeOverrideTest.java

```diff
@@ -0,0 +1,191 @@
+package com.fasterxml.jackson.databind.ser.filter;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.databind.BaseMapTest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import static junit.framework.TestCase.assertEquals;
+
+/**
+ * Unit tests for checking that overridden settings for
+ * {@link JsonSerialize#include} annotation property work
+ * as expected.
+ */
+public class JsonIncludeOverrideTest
```

Comment:
Perhaps it would make sense to create a separate test class to contain new tests? Not sure which makes most sense: these are of course related, just trying to find good balance between huge test classes and big number of classes.